src/runtime/providers/sirv.ts

```diff
@@ -0,0 +1,137 @@
+import { joinURL } from 'ufo'
+import type { ProviderGetImage } from '../../types'
+import { createOperationsGenerator } from '#image'
+const deleteHash = (value: string) => value.startsWith('#') ? value.replace('#', '') : value
+const generateColorKeys = () => {
+  const keysNeedingHashDeletion = [
+    'canvasBorderColor',
+    'frameColor',
+    'frameRimColor',
+    'colorizeColor',
+    'colortoneColor',
+    'textColor',
+    'textoutlineColor',
+    'textBackgroundColor'
+  ]
+
+  return Object.fromEntries(
+    keysNeedingHashDeletion.map(key => [key, (value: string) => deleteHash(value)])
+  )
+}
+export const operationsGenerator = createOperationsGenerator({
+  keyMap: {
+    width: 'w',
+    height: 'h',
+    s: 's',
+    quality: 'q',
+    fit: 'scale.option',
+    profile: 'profile',
+    format: 'format',
+    webpFallback: 'webp-fallback',
+    subsampling: 'subsampling',
+    gifCompression: 'gif.lossy',
+    crop: 'crop.type',
+    cropAr: 'crop.aspectratio',
+    cw: 'cw',
+    ch: 'ch',
+    cx: 'cx',
+    cy: 'cy',
```

Comment:
you can omit the keys that are equal to their values; the generator will just return the key if there's not an entry in the key map.

https://stackblitz.com/edit/github-z7xwz3?file=nuxt.config.ts,app.vue

Reply:
I see, thanks, addressed in https://github.com/nuxt/image/pull/1066/commits/d876277f5585b78b220ed68d0ce7c2b0c1c41cdf